### PR TITLE
doc: update cephfs snapshot, restore and clone support matrix

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

updates: #1242

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,9 +75,12 @@ NOTE:
 | CephFS | Dynamically provision, de-provision File mode RWO volume  | Beta           | >= v1.1.0          | >= v1.0.0        | Nautilus (>=14.2.2)  | >= v1.14.0         |
 |        | Dynamically provision, de-provision File mode RWX volume  | Beta           | >= v1.1.0          | >= v1.0.0        | Nautilus (>=v14.2.2) | >= v1.14.0         |
 |        | Dynamically provision, de-provision File mode ROX volume  | Alpha          | >= v3.0.0          | >= v1.0.0        | Nautilus (>=v14.2.2) | >= v1.14.0         |
-|        | Creating and deleting snapshot                            | -              | -                  | -                | -                    | -                  |
-|        | Provision volume from snapshot                            | -              | -                  | -                | -                    | -                  |
-|        | Provision volume from another volume                      | -              | -                  | -                | -                    | -                  |
+|        | Creating and deleting snapshot                            | Alpha          | >= v3.1.0          |  >= v1.0.0       | Nautilus (>v14.2.10) | >= v1.17
```
Comment (Madhu-1) [inline README.md:78]: do we support snapshot and cloning in v14.2.10 :thinking:

Comment (humblec) [reply README.md:78]: madhu didnt I mention >14.2.10 ?

Comment (Madhu-1) [reply README.md:78]: sorry, I missed it. but still, as we have not tested with Nautilus isn't it good to mention Octopus?

Comment (humblec) [reply README.md:78]: yeah, that can be done too.. :+1:

Comment (humblec) [reply README.md]: @Madhu-1 mentioned octopus.. ptal.

File: README.md [hunk 2]
```diff
@@ -75,9 +75,12 @@ NOTE:
 | CephFS | Dynamically provision, de-provision File mode RWO volume  | Beta           | >= v1.1.0          | >= v1.0.0        | Nautilus (>=14.2.2)  | >= v1.14.0         |
 |        | Dynamically provision, de-provision File mode RWX volume  | Beta           | >= v1.1.0          | >= v1.0.0        | Nautilus (>=v14.2.2) | >= v1.14.0         |
 |        | Dynamically provision, de-provision File mode ROX volume  | Alpha          | >= v3.0.0          | >= v1.0.0        | Nautilus (>=v14.2.2) | >= v1.14.0         |
-|        | Creating and deleting snapshot                            | -              | -                  | -                | -                    | -                  |
-|        | Provision volume from snapshot                            | -              | -                  | -                | -                    | -                  |
-|        | Provision volume from another volume                      | -              | -                  | -                | -                    | -                  |
+|        | Creating and deleting snapshot                            | Alpha          | >= v3.1.0          |  >= v1.0.0       | Octopus (>=v15.2.3)  | >= v1.17
```
Comment (Madhu-1) [inline README.md:78]: don't we have functionality in 15.0.0?

Comment (humblec) [reply README.md:78]: snap info is after v15.2.2+ is what I am seeing.

**Comment by humblec** [comment]
@Madhu-1 can you add `ready-to-merge` here please ?

File: README.md [hunk 3]
```diff
@@ -75,9 +75,12 @@ NOTE:
...
+0         |
```
Comment (Yuggupta27) [inline README.md:79]: Is the `0` here intentional?

Comment (Yuggupta27) [reply README.md:79]: Seems to be coming in the `plugin` section

Comment (Madhu-1) [reply README.md:79]: @Yuggupta27 Thanks for catching this @humblec cephfs matrix is broken https://github.com/ceph/ceph-csi/blob/68b56e260f98fb4ba495d090e23780d50c18875b/README.md#support-matrix

Comment (humblec) [reply README.md:79]: Thanks @Yuggupta27 . corrected it :+1:

**Comment by humblec** [comment]
@Madhu-1 corrected it . PTAL :)

File: README.md (hunk 3 above)
Comment (Madhu-1) [reply README.md]: still, I see the same issue

Comment (humblec) [reply README.md]: @Madhu-1 can you referesh

**Comment by Madhu-1** [comment]
@MergifyIO rebase

**Comment by humblec** [comment]
@mergify rebase

**Comment by humblec** [comment]
@Mergify rebase

**Comment by Madhu-1** [comment]
@mergifyio rebase

**Pull request merged**